multiqc/modules/sourmash/gather.py

```diff
@@ -0,0 +1,221 @@
+#!/usr/bin/env python
+
+""" MultiQC module to parse similarity matrix output by sourmash gather """
+
+import csv
+import logging
+from collections import OrderedDict
+
+from multiqc.plots import bargraph
+
+# Initialise the logger
+log = logging.getLogger(__name__)
+
+
+class GatherMixin:
+    def parse_gather(self):
+        """
+        Modeled after the kraken module and the sourmash compare module.
+        """
+
+        self.top_n = 5
```

Comment:
Might be nice to allow the end user to configure this?

```suggestion
        self.top_n = getattr(config, 'sourmash', {}).get('gather', {}).get('top_n', 5)
```

Would allow the end user to set this in a config:

```yaml
sourmash:
    gather:
        top_n: 8
```

(please check syntax, typed quickly in a web browser)

Reply:
In this case, we should do the same for Kraken. I'll make a separate PR.